vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs

```diff
@@ -18,53 +18,102 @@ open Microsoft.VisualStudio.FSharp.Editor.Structure
 
 module internal BlockStructure =
     let scopeToBlockType = function
-    | Scope.Open -> BlockTypes.Imports
-    | Scope.Namespace
-    | Scope.Module -> BlockTypes.Namespace 
-    | Scope.Record
-    | Scope.Interface
-    | Scope.TypeExtension
-    | Scope.RecordDefn
-    | Scope.CompExpr
-    | Scope.ObjExpr
-    | Scope.UnionDefn
-    | Scope.Attribute
-    | Scope.Type -> BlockTypes.Type
-    | Scope.New
-    | Scope.RecordField
-    | Scope.Member -> BlockTypes.Member
-    | Scope.LetOrUse
-    | Scope.Match
-    | Scope.MatchClause
-    | Scope.EnumCase
-    | Scope.UnionCase
-    | Scope.MatchLambda
-    | Scope.ThenInIfThenElse
-    | Scope.ElseInIfThenElse
-    | Scope.TryWith
-    | Scope.TryInTryWith
-    | Scope.WithInTryWith
-    | Scope.TryFinally
-    | Scope.TryInTryFinally
-    | Scope.FinallyInTryFinally
-    | Scope.IfThenElse-> BlockTypes.Conditional
-    | Scope.Tuple
-    | Scope.ArrayOrList
-    | Scope.CompExprInternal
-    | Scope.Quote
-    | Scope.SpecialFunc
-    | Scope.Lambda
-    | Scope.LetOrUseBang
-    | Scope.Val
-    | Scope.YieldOrReturn
-    | Scope.YieldOrReturnBang
-    | Scope.TryWith -> BlockTypes.Expression
-    | Scope.Do -> BlockTypes.Statement
-    | Scope.While
-    | Scope.For -> BlockTypes.Loop
-    | Scope.HashDirective -> BlockTypes.PreprocessorRegion
-    | Scope.Comment
-    | Scope.XmlDocComment -> BlockTypes.Comment
+        | Scope.Open -> BlockTypes.Imports
+        | Scope.Namespace
+        | Scope.Module -> BlockTypes.Namespace 
+        | Scope.Record
+        | Scope.Interface
+        | Scope.TypeExtension
+        | Scope.RecordDefn
+        | Scope.CompExpr
+        | Scope.ObjExpr
+        | Scope.UnionDefn
+        | Scope.Attribute
+        | Scope.Type -> BlockTypes.Type
+        | Scope.New
+        | Scope.RecordField
+        | Scope.Member -> BlockTypes.Member
+        | Scope.LetOrUse
+        | Scope.Match
+        | Scope.MatchClause
+        | Scope.EnumCase
+        | Scope.UnionCase
+        | Scope.MatchLambda
+        | Scope.ThenInIfThenElse
+        | Scope.ElseInIfThenElse
+        | Scope.TryWith
+        | Scope.TryInTryWith
+        | Scope.WithInTryWith
+        | Scope.TryFinally
+        | Scope.TryInTryFinally
+        | Scope.FinallyInTryFinally
+        | Scope.IfThenElse-> BlockTypes.Conditional
+        | Scope.Tuple
+        | Scope.ArrayOrList
+        | Scope.CompExprInternal
+        | Scope.Quote
+        | Scope.SpecialFunc
+        | Scope.Lambda
+        | Scope.LetOrUseBang
+        | Scope.Val
+        | Scope.YieldOrReturn
+        | Scope.YieldOrReturnBang
+        | Scope.TryWith -> BlockTypes.Expression
+        | Scope.Do -> BlockTypes.Statement
+        | Scope.While
+        | Scope.For -> BlockTypes.Loop
+        | Scope.HashDirective -> BlockTypes.PreprocessorRegion
+        | Scope.Comment
+        | Scope.XmlDocComment -> BlockTypes.Comment
+
+    let isAutoCollapsible = function
```

Comment:
@vasily-kirichenko 
Just a code formatting thing ... the first two times I looked at this I thought that the only return value was false.  Perhaps if we put a blank line between the true and false blocks I would have caught the trues earlier.  But perhaps that is just me.

Kevin